metrics/mapping.pl

```diff
@@ -1189,50 +1197,6 @@
   ],
 };
 
-$map{cmss10} = {
```

Comment:
I just noticed that this appears twice so I removed the one that didn't match makeFF.